lib/file_mutex.rb

```diff
@@ -24,6 +24,9 @@ class FileMutex
   # `Timeout::Error`.
   DEFAULT_TIMEOUT = 60.minutes
 
+  # The interval between retries.
+  RETRY_INTERVAL_IN_SECONDS = 1
```

Comment:
just say `RETRY_INTERVAL = 1.second`, the value will then include its unit

Reply:
Done.